### ISSUE TITLE
[a11y Design Review] Core Identity Team

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#62536]
## Auditor(s)
Eli 

## Design Review
Based on the development of the prototype, some of the following may or may not be applicable. 
- [ ] [Color and contrast checks](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#use-of-color)
  - [ ] WCAG 2.1 AA
  - [ ] WCAG 2.1 AAA
  - [ ] [Who Can Use](https://www.whocanuse.com/?b=ecece5&c=000000&f=20&s=)
  - [ ] High Contrast Mode
- [ ] [Keyboard navigation](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#keyboard-nav)
  - [ ] Annotations for Tab order, reflow and reading order (if needed)
- [ ] Accessible Name Annotations
  - [ ] [Alt attributes for images](https://webaim.org/techniques/alttext/)
  - [ ] Text alternative for complex images such as charts graphs maps etc.
- [ ] Language (ease of reading)
- [ ] HTML Tags
  - [ ] [Page Title](https://accessibility.18f.gov/page-titles/)
  - [ ] [Heading order](https://accessibility.18f.gov/headings/)
   - [ ] Annotations (if needed)
  - [ ] Semantic Consideration for paragraphs ordered lists numbering
  - [ ] [Tables (heading rows)](https://design.va.gov/components/table)
- [ ] Size of clickable areas

### As Time Allows
- [ ] [Components](https://design.va.gov/components/) follow VA design system guidance.
- [ ] [Patterns](https://design.va.gov/patterns/) follow VA design system guidance.

## Prototype Link

[Link to prototype](insert link here)

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- If no research has been conducted with assistive technology users, it should be a priority

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- When possible, point to more research with assistive technology users

### Concise issue description
- Where you found it
- More details

#### Bandaid solution
- If helpful, provide a temporary solution that involves less content/design changes

#### Recommended solution
- Try to provide an ideal solution too that explores alternatives should content, design, IA, the broader service, etc can be fixed
- When possible, point to more research with assistive technology users

**Comment by eli-oat** [comment]
## NOTES

I will update the ticket with the final info when I'm done with my review: 

- this is high-level because we're only working with Figma mocks at this point
- my main concern is with the internal-scrolling section that contains the term of service content
    - does this have to scroll? is this the guidance of IA?
    - does the terms of service content follow plain-language guidance? 
        - is there an opportunity to have a shorter, explainer at the top? (validate with Content/IA)
    - if you do go with an internal-scrolling section we will need to be careful with how focus is managed in and around it, especially on mobile

**Comment by sara-amanda** [comment]
## CAIA Button Note

### c/o Clayton

- [Convo with Clayton and Laura via Slack (7/25/2023)](https://dsva.slack.com/archives/C01K37HRUAH/p1690301563909669?thread_ts=1690299277.325729&cid=C01K37HRUAH)
- Our team discussed things late last week after we had a chance to look at the [latest version of the terms of use flow your team worked on with stakeholders](https://dvagov-my.sharepoint.com/:w:/g/personal/danielle_thierry_va_gov/EUJ2CL4M4QZHvtC036A9dMkBSAskrTEgRYAFC40dlpzRtQ?e=HqzNbC). Since we are following that language, we didn’t think we needed to extend the content discussion overall.
- The exception is the `“don’t accept”` button, which we will test as `“decline”` in our UX research and see how that plays before finalizing.

[Ref. UX Research Ticket #62359 for Core Identity Team](https://app.zenhub.com/workspaces/sitewide-content-accessibility-and-ia-63a1d63232beba0011a7833f/issues/gh/department-of-veterans-affairs/va.gov-team/62359)

**Comment by eli-oat** [comment]
@claytonzook CAIA has finished an initial review of what was provided to us. Details in the ticket. 

Please don't hesitate to reach out here or on Slack for further info or input/clarifications. 

cc @sara-amanda
